### ISSUE TITLE
"Error: unknown flag: --override-variant" - required for fetching arm/v7 images

### ISSUE THREAD

**Issue by srd424** [containers/podman#7423]
**Is this a BUG REPORT or FEATURE REQUEST? (leave only one on its own line)**

/kind bug

I'm trying do download the arm/v7 version of docker://docker.io/ubuntu:focal on my amd64 development machine, unfortunately I get:

```
$ podman pull --override-arch arm --override-variant v7  docker://docker.io/ubuntu:focal
Error: unknown flag: --override-variant
```

Weirdly, --override-variant is suggestion by the tab-autocomplete feature in bash!

**Steps to reproduce the issue:**

1. Run `podman pull --override-arch arm --override-variant v7  docker://docker.io/ubuntu:focal` on an amd64 machine.

**Describe the results you received:**

`Error: unknown flag: --override-variant`

**Describe the results you expected:**

For the image to download successfully.

**Additional information you deem important (e.g. issue happens only occasionally):**

My development environment is a xubuntu bionic nspawn container.

**Output of `podman version`:**

```
Version:      2.0.4
API Version:  1
Go Version:   go1.14.4
Built:        Thu Jan  1 01:00:00 1970
OS/Arch:      linux/amd64
```

**Output of `podman info --debug`:**

```
host:
  arch: amd64
  buildahVersion: 1.15.0
  cgroupVersion: v1
  conmon:
    package: 'conmon: /usr/libexec/podman/conmon'
    path: /usr/libexec/podman/conmon
    version: 'conmon version 2.0.20, commit: '
  cpus: 4
  distribution:
    distribution: ubuntu
    version: "18.04"
  eventLogger: file
  hostname: xubuntu
  idMappings:
    gidmap:
    - container_id: 0
      host_id: 1000
      size: 1
    - container_id: 1
      host_id: 100000
      size: 65536
    uidmap:
    - container_id: 0
      host_id: 1000
      size: 1
    - container_id: 1
      host_id: 100000
      size: 65536
  kernel: 5.4.0-42-generic
  linkmode: dynamic
  memFree: 1804713984
  memTotal: 16666976256
  ociRuntime:
    name: runc
    package: 'runc: /usr/sbin/runc'
    path: /usr/sbin/runc
    version: 'runc version spec: 1.0.1-dev'
  os: linux
  remoteSocket:
    path: /run/user/1000/podman/podman.sock
  rootless: true
  slirp4netns:
    executable: /usr/bin/slirp4netns
    package: 'slirp4netns: /usr/bin/slirp4netns'
    version: |-
      slirp4netns version 1.1.4
      commit: unknown
      libslirp: 4.3.1-git
      SLIRP_CONFIG_VERSION_MAX: 3
  swapFree: 4602404864
  swapTotal: 4689227776
  uptime: 239h 58m 2.73s (Approximately 9.96 days)
registries:
  search:
  - docker.io
  - quay.io
store:
  configFile: /home/steved/.config/containers/storage.conf
  containerStore:
    number: 0
    paused: 0
    running: 0
    stopped: 0
  graphDriverName: overlay
  graphOptions:
    overlay.mount_program:
      Executable: /usr/local/bin/fuse-overlayfs
      Package: Unknown
      Version: |-
        fuse-overlayfs: version 1.1.0
        FUSE library version 3.9.0
        using FUSE kernel interface version 7.31
  graphRoot: /home/steved/.local/share/containers/storage
  graphStatus:
    Backing Filesystem: extfs
    Native Overlay Diff: "false"
    Supports d_type: "true"
    Using metacopy: "false"
  imageStore:
    number: 8
  runRoot: /run/user/1000/containers
  volumePath: /home/steved/.local/share/containers/storage/volumes
version:
  APIVersion: 1
  Built: 0
  BuiltTime: Thu Jan  1 01:00:00 1970
  GitCommit: ""
  GoVersion: go1.14.4
  OsArch: linux/amd64
  Version: 2.0.4
```

**Package info (e.g. output of `rpm -q podman` or `apt list podman`):**

```
Listing... Done
podman/unknown,now 2.0.4~1 amd64 [installed]
```

**Have you tested with the latest version of Podman and have you checked the Podman Troubleshooting Guide?**

Yes

**Additional environment details (AWS, VirtualBox, physical, etc.):**

**Comment by srd424** [comment]
Workaround using skopeo:
`skopeo  --override-arch arm --override-variant v7 copy  docker://docker.io/ubuntu:focal containers-storage:docker.io/library/ubuntu:focal-armhf`

**Comment by QiWang19** [comment]
Close the issue since it's fixed by #7436
